.storybook/config.js

```diff
@@ -1,47 +1,29 @@
 /* eslint-disable no-restricted-globals */
 /* global __STORYBOOK_CLIENT_API__ */
+
 /**
  * External dependencies
  */
+import lodash from 'lodash';
 import React from 'react';
 import { addDecorator, configure } from '@storybook/react';
 
 /**
  * WordPress dependencies
  */
 import { createHigherOrderComponent } from '@wordpress/compose';
-import {
-	Component,
-	createRef,
-	Fragment,
-	createElement,
-	createPortal,
-} from '@wordpress/element';
+import { Component, createRef, Fragment, createElement, createPortal } from '@wordpress/element';
 import { __, sprintf, setLocaleData } from '@wordpress/i18n';
-import {
-	getQueryString,
-	addQueryArgs,
-} from '@wordpress/url';
-import lodash from 'lodash';
-import {
-	addFilter,
-	removeFilter,
-	addAction,
-	doAction,
-	applyFilters,
-	removeAction,
-	removeAllFilters,
-} from '@wordpress/hooks';
+import { getQueryString, addQueryArgs } from '@wordpress/url';
+import { addFilter, removeFilter, addAction, doAction, applyFilters, removeAction, removeAllFilters } from '@wordpress/hooks';
+
```

Comment:
It looks like the import format was only changed from one-per-line to all on one line here. Can we keep it as it was so it's easier to see what changed? It's also more consistent with how we format multiple imports in other components.

Reply:
> It's also more consistent with how we format multiple imports in other components.

We don't have consistency even in the original version of this file (compare imports from `@wordpress/i18n` and other WP dependencies). There is a nice eslint rule which can help us to make it consistent: `object-curly-newline`. We can use it like to restrict how many properties we allow to import using a single-line approach and when to start using multiline. For example, something like this:

```
"object-curly-newline": [ "error", { "ImportDeclaration": "always", "minProperties": 4 } ],
```
> Can we keep it as it was so it's easier to see what changed?

I'll revert.

Reply:
Updated.